examples/lammps/h2o-cavmd/README.md

```diff
@@ -36,4 +38,4 @@ In short, to perform CavMD simulations, we need to make **two modifications** of
        L -8.65101e+00  1.11541e+00  4.56823e-01
        L  5.35376e-01  1.20389e+01 -1.19497e-01
 ```
-Here, we have 216 water molecules (648 nuclei) and two photons (labeled as **L**). Please always put the photons at the very end of the xyz file. The mass of each photon is 1.0 atomic units by default. The first photon is coupled to the x direction of the total dipole moment of water, and the second photon is coupled to the y direction. Hence, for the first photon "atom", only the x coordinate is influenced by the molecules; for the second photon "atom", only the y coordinate is influenced by the molecules.
+Here, we have 216 water molecules (648 nuclei) and two photons (labeled as **L**). Please always put the photons at the very end of the xyz file. The mass of each photon is 1.0 atomic units by default. We assume a z-direction oriented optical cavity coupled to the molecules, so the k-vector of the cavity photons is along the z-direction, and the cavity photons can polarize along either the x- or y-direction. The first photon is coupled to the x direction of the total dipole moment of water, and the second photon is coupled to the y direction. Hence, for the first photon "atom", only the x coordinate is influenced by the molecules; for the second photon "atom", only the y coordinate is influenced by the molecules. In the current implementation, only two cavity photons are supported (more advanced cavity setups will be included in an updated version of the code).
```

Comment:
Please add a soft exit with this message in case the user use tries to use >2 photons.